core/src/main/scala/org/apache/spark/api/r/RBackendHandler.scala

```diff
@@ -148,6 +151,9 @@ private[r] class RBackendHandler(server: RBackend)
       case e: Exception =>
         logError(s"$methodName on $objId failed", e)
```

Comment:
Can we remove the `e` from this `logError` call ? Because right now we get the exception printed twice [1]. Also is there a max length for the string printed by R ? If you see the gist we right not get only the first 10 or so lines from the exception -- this might be fine, but I just wanted to check.

[1] https://gist.github.com/shivaram/e57e83f2b77e369796c8